docs/source/_templates/sphinx_conf.template.py

```diff
@@ -37,9 +38,10 @@
     "sphinx.ext.mathjax",
     "sphinx.ext.napoleon",
     "sphinx.ext.viewcode",
-    "sphinxarg.ext",
```

Comment:
Is this unused?

I wonder if any of the other plugins are unused.

Reply:
These plugins are not in use:

![image](https://github.com/user-attachments/assets/e74c5f6c-a791-49fb-978a-509ea5350043)

Reply:
I'll remove these in a separate PR. Thank you, @jsolbrig for pointing this out!

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/853